src/tcgwars/logic/impl/gen7/BurningShadows.groovy

```diff
@@ -1310,7 +1308,6 @@ public enum BurningShadows implements LogicCardInfo {
               def list = opp.hand.shuffledCopy().showToMe("Opponent's hand").filterByType(BASIC)
               if(list){
                 def card = list.select("Put a Basic Pokémon you find there onto your opponent's Bench").first()
-                opp.hand.remove(card)
                 def pcs = benchPCS(card, OTHER, TargetPlayer.OPPONENT)
                 directDamage(30, pcs, SRC_ABILITY)
```

Comment:
Add condition pls

Reply:
Added conditions here and in similar usages